.gitignore

```diff
@@ -1,3 +1,5 @@
 # PyCache files
 __pycache__/
-cutlass_library.egg-info/
\ No newline at end of file
+cutlass_library.egg-info/
+build/
+.vscode/
```

Comment:
I'm hoping this is okay to add? I've a habit of just doing `git add .` for small repos and it can get annoying having to deal with me accidently committing my build directory.